### PR TITLE
[Ogoyi]-Sage-FloralNft Smart Contract for Buying and Gifting Flowers as NFTs

### PR DESCRIPTION
**Pull request by Ogoyi**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-3-23-floral-tutorial-sage/floral-tutorial-sage.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+authors:
+  - name: Ogoyi Thompson
+    title: Technical Writer
+url: https://github.com/Ogoyi
+tags: [solidity, intermediate, celo]
+hide_table_of_contents: true
+slug: /tutorials/FloralNft-Smart-Contract-for-Buying-and-Gifting-Flowers-as-NFTs
+---
+
+![header](../../src/data-tutorials/showcase\intermediate\FloralNft-Smart-Contract-for-Buying-and-Gifting-Flowers-as-NFTs.png.png)
```
Comment (Ishanpathak1) [inline blog/03-3-23-floral-tutorial-sage/floral-tutorial-sage.md:13]: Slash are in wrong directions

File: blog/03-3-23-floral-tutorial-sage/floral-tutorial-sage.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+
+
+ # A Detailed Guide on How to Create a Smart Contract for a Flowers Marketplace on the Celo Blockchain
```
Comment (Ishanpathak1) [inline blog/03-3-23-floral-tutorial-sage/floral-tutorial-sage.md:16]: We dont need this as we will be getting title from metadata

**Review by Ishanpathak1 — COMMENTED** [review]
Can you make the required changes, and let me know after you make the changes

File: blog/03-3-23-floral-tutorial-sage/floral-tutorial-sage.md [hunk 3]
```diff
@@ -0,0 +1,401 @@
+---
+title: FloralNft Smart Contract for Buying and Gifting Flowers as NFTs.
+description: FloralNft is a smart contract built on the Celo blockchain that allows users to create and trade unique digital flowers as non-fungible tokens (NFTs).
+authors:
+  - name: Ogoyi Thompson
+    title: Technical Writer
+url: https://github.com/Ogoyi
+tags: [solidity, intermediate, celo]
```
Comment (Ishanpathak1) [inline blog/03-3-23-floral-tutorial-sage/floral-tutorial-sage.md:8]: Add celo sage as a tag

File: src/data-tutorials/users.tsx [hunk 4]
```diff
@@ -912,6 +912,13 @@ const Users: User[] = [
     website: 'blog/tutorials/flutter-celo-easily-build-flutter-mobile-dApps',
     tags: ['intermediate','foundation', 'flutter', 'composer'],
   },
+  {
+    title: 'FloralNft Smart Contract for Buying and Gifting Flowers as NFTs',
+    description: 'FloralNft is a smart contract built on the Celo blockchain that allows users to create and trade unique digital flowers as non-fungible tokens (NFTs).',
+    preview: require('./showcase/intermediate/FloralNft-Smart-Contract-for-Buying and-Gifting-Flowers as-NFTs.png'),
+    website: 'blog/tutorials/FloralNft-Smart-Contract-for-Buying-and-Gifting-Flowers-as-NFTs',
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:919]: Sluggify this,i.e. no capital letters

**Review by Ishanpathak1 — COMMENTED** [review]
Make the changes and let me know

File: src/data-tutorials/users.tsx [hunk 5]
```diff
@@ -912,6 +912,13 @@ const Users: User[] = [
     website: 'blog/tutorials/flutter-celo-easily-build-flutter-mobile-dApps',
     tags: ['intermediate','foundation', 'flutter', 'composer'],
   },
+  {
+    title: 'FloralNft Smart Contract for Buying and Gifting Flowers as NFTs',
+    description: 'FloralNft is a smart contract built on the Celo blockchain that allows users to create and trade unique digital flowers as non-fungible tokens (NFTs).',
+    preview: require('./showcase/intermediate/FloralNft-Smart-Contract-for-Buying and-Gifting-Flowers as-NFTs.png'),
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:918]: Missing - in some part -./showcase/intermediate/FloralNft-Smart-Contract-for-Buying-and-Gifting-Flowers-as-NFTs.png

**Review by Ishanpathak1 — COMMENTED** [review]
Please make the required changes in the users.tsx file

File: src/data-tutorials/users.tsx [hunk 6]
```diff
@@ -912,6 +912,13 @@ const Users: User[] = [
     website: 'blog/tutorials/flutter-celo-easily-build-flutter-mobile-dApps',
     tags: ['intermediate','foundation', 'flutter', 'composer'],
   },
+  {
+    title: 'FloralNft Smart Contract for Buying and Gifting Flowers as NFTs',
+    description: 'FloralNft is a smart contract built on the Celo blockchain that allows users to create and trade unique digital flowers as non-fungible tokens (NFTs).',
+    preview: require('./showcase/intermediate/FloralNft-Smart-Contract-for-Buying-and-Gifting-Flowers-as-NFTs.png'),
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:918]: add .png as your image name has two .png, or change the name of the image and make sure you change in the header of .md file too

**Review by Ishanpathak1 — COMMENTED** [review]
These are the last changes after that we will be moving this in publish section

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**